index.test.js

```diff
@@ -410,7 +410,7 @@ describe('flags use-typography-styles', () => {
 // only vertical margins are allowed for non-flexible spacing
 const unitsUsageCSS = `.valid-usage {
   margin-top: 20px;
-  margin-top: 10px;
```

Comment:
This flagged `declaration-block-no-duplicate-properties`, which went against what was being tested. 

Instead make it pass the test by changing it to a different value, as the recommended rule allows this: https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties/#ignore-consecutive-duplicates-with-different-values